docs/api/portable-stories/portable-stories-vitest.mdx

```diff
@@ -170,25 +170,18 @@ sidebar:
   ## setProjectAnnotations
 
   This API should be called once, before the tests run, typically in a [setup file](https://vitest.dev/config/#setupfiles). This will make sure that whenever `composeStories` or `composeStory` are called, the project annotations are taken into account as well.
-  
+
   These are the configurations needed in the setup file:
   - preview annotations: those defined in `.storybook/preview.ts`
   - addon annotations (optional): those exported by addons
   - beforeAll: code that runs before all tests ([more info](../../writing-tests/interaction-testing.mdx#beforeall))
-  - testingLibraryRender: the render function from Testing Library
 
   {/* prettier-ignore-start */}
 
   <CodeSnippets path="portable-stories-vitest-set-project-annotations.md" />
 
   {/* prettier-ignore-end */}
 
-  <Callout variant="warning">
-
-  You **must** provide `testingLibraryRenderer` to use portable stories with Vitest.
-
-  </Callout>
-
   {/* TODO: Create issue for interest in non-Testing Library render option, with recipe, and mention here (Jest, too) */}
```

Comment:
```suggestion
```

Reply:
We should also consider closing the [issue](https://github.com/storybookjs/storybook/discussions/28532)?

It's had zero engagement.